internal/transport/transport.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2022 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package transport
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/crossplane/crossplane/internal/version"
+)
+
+// DefaultUserAgent is the default User-Agent header that is set when making
+// HTTP requests for packages.
+var DefaultUserAgent = fmt.Sprintf("%s/%s", "crossplane", version.New().GetVersionString())
```

Comment:
Just to double-check, it should also expose UXP in the form of `up` in the version like `1.9.1-up.2` is that correct?

Reply:
@ytsarev that will be a downstream concern 👍🏻